apps/modernization-ui/src/apps/search/advancedSearch/AdvancedSearch.scss

```diff
@@ -1,5 +1,15 @@
 @use 'styles/colors';
 
+.advanced-search-title {
+    font-size: 100%;
+}
+
+.advanced-search-results-title {
+    margin-block-start: 0 !important;
+    margin-block-end: 0 !important;
+    font-size: 1.13rem;
```

Comment:
I think `h2` should have a height of `1.375rem`

Reply:
Copied the 1.13rem to keep the current font-size the same. I can make change if H2 should be uniform.  

![Screenshot 2024-01-30 at 6 21 49 PM](https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/e48276c1-69b6-432f-a040-dae4131ddf60)